robot_skills/src/robot_skills/arms.py

```diff
@@ -16,7 +16,7 @@
 from robot_part import RobotPart
 
 
-class ArmState:
+class GripperState:
```

Comment:
This breaks the API, are you sure you checked all relevant packages that might use this class?

Reply:
Yep, checked all packages. Only used in two files, which I changed.